### PR TITLE
Add a docker hypervisor

### PR DESCRIPTION
**Pull request by ekohl**
This also copies in the docker image commands as used by Voxpupuli.

### CODE REVIEW

File: lib/beaker-hostgenerator/data.rb [hunk 1]
```diff
@@ -270,6 +276,11 @@ def osinfo
             'platform'           => 'debian-7-amd64',
             'packaging_platform' => 'debian-7-amd64'
           },
+          :docker => {
+            'docker_image_commands': [
+              'apt-get update && apt-get install -y cron locales-all net-tools wget'
```
Comment (mchllweeks) [inline lib/beaker-hostgenerator/data.rb]: @ekohl Looks like a typo here where ':' should actually be '=>'

Comment (ekohl) [reply lib/beaker-hostgenerator/data.rb]: Fixed

**Comment by kevpl** [comment]
This seems fine to me. @puppetlabs/qe-team any concerns about merging/releasing these changes?

**Comment by tvpartytonight** [comment]
Hey @ekohl thanks for your contribution; do you think that you could add a test for this? There's not quite an obvious place to put it, but if you add a machine with the docker hypervisor [here](https://github.com/puppetlabs/beaker-hostgenerator/blob/master/test/fixtures/per-host-settings/every-hypervisor.yaml) I'd think that would be sufficient. Thanks!

**Comment by ekohl** [comment]
@tvpartytonight updated with a test.

**Comment by ekohl** [comment]
Anything else I should add?

**Pull request merged**